setController.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#Requires one parm - controller version
```

Comment:
Can you remove this comment? The script doesn't take any parms.

Reply:
OK NP